App/CourseApp/SharedUI/UIFont+Ext.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  UIFont+Ext.swift
```

Comment:
It's subjective but this name is not very self-describing what the extension does. Better to have multiple extensions with proper naming like Font+AppFonts, Font+Autoloading etc